app/src/Homepage/Homepage.css

```diff
@@ -46,3 +46,78 @@
   height: 200vh;
   text-align: center;
 }
+
+/* Welcome header */
+.welcome {
+  text-align: center;
+  margin-top: 60px;
+  font-size: 300%;
+}
+
+/* How it works: header */
+.howitworks {
+  position: absolute;
+  top: 15%;
+  height: 50%;
+  width: 100%;
+}
+
+/* Explanation below the How it works: header */
+.explanation {
+  font-weight: normal;
+}
+
+/* The rating boxes for difficulty, workload, and practicality */
+.ratingbox {
+  font-weight: bold;
+  font-size: 130%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  position: absolute;
+  margin-top: 20px;
+  left: 10%;
+  height: 10%;
+  width: 14%;
+}
+
+.ratingbox-diff {
+  display: flex;
+  background-color: #e087e2;
+}
+
+.ratingbox-work {
+  background-color: #8dbbff;
+  left: 43%;
```

Comment:
Have you thought about wrapping the RatingBoxes and RatingDescriptions in div's with the same flexbox css?

Like, instead of these hard-coded 'left' fields, define a wrapper with 'display: flex' and 'justify-content: space-around'?

Reply:
Omg yeah I came across flexbox when I was doing this but I wasn't sure if I should do it. I'll play around with that now

Reply:
Fixed